x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -351,6 +351,11 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
     ]
   );
 
+  const getCellActionsStub = useMemo(
+    () => ({ getCellActions: () => null, visibleCellActions: undefined, disabledCellActions: [] }),
+    []
+  );
+
```

Comment:
Consider moving this to outside the component e.g. just after `isSystemCell`. No need to memoize a constant :)

Reply:
Done.